### DISCUSSION TITLE
What about automatic sitemap.xml generation?

### DISCUSSION THREAD

**Discussion by strzibny** [sitepress/sitepress#52] [General]
I wonder if Sitepress could automatically generate a sitemap file or at least pregenerate a sitemap.xml.erb template perhaps? While it's fairly easy I feel like something like sitemap should be everywhere from the start.

**Comment by MSchmidt** [comment]
@strzibny  Did you end up creating one yourself? I'm in need of a sitemap now and am not sure what the best approach would be. Either create a fully manual? Or just looping over all pages with a glob? Usually they have a change date, which I do not have available I guess.

**Reply by strzibny** [reply]
I just made my own with a glob...

**Reply by MSchmidt** [reply]
Would you consider sharing the file? Would be much appreciated.
